websites/P/Paramount+/presence.ts

```diff
@@ -30,15 +30,15 @@ const presence = new Presence({
       elapsed = Math.floor(Date.now() / 1000);
     }
 
-    presenceData.state = undefined;
+    delete presenceData.state;
```

Comment:
Like I said before, you are doing this for nothing so it this line can be deleted